server/routes/urlshortener.js

```diff
@@ -32,12 +34,8 @@ module.exports = app => {
             return res.status(400).send("error");
         }
 
-        // if an url:
-        let index = getSmallestIndex();
-        keys.push(index); // TODO: This might be removed
-
-        var shortenedUrlId = code.encode(index);
-        const shortBaseUrl = WINDOW_LOCATION_ORIGIN + ':' + PORT;
+        let shortenedUrlId = getAutoincrementedId();
+        const shortBaseUrl = WINDOW_LOCATION_ORIGIN + ':' + PORT; // TODO: Why would this be necessary?
```

Comment:
It just is a field of the class that returns an instance of this class, where the baseURL + id is constructed. So when a user requests its urls to be shortened. this complete url is returned to the user

Reply:
What I meant with this is, that it does not make sense to add `WINDOW_LOCATION_ORIGIN + ':' + PORT` because it is implicitliy given at all times, when a API server is called for a certain adress. I does not add any value nor it necessary at all to provide the functionality of the RESTApi

Reply:
Sure, the consumer of the api can concatenate these values themselves.